scripts/check_configs.ts

```diff
@@ -596,6 +675,16 @@ async function main() {
         outcomeMap[network] = false;
       }
 
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
+        const polynetworkBridges = bridgesMap.polynetwork.map((bridge) => bridge.bridgeAddresses).flat()
+        const ibcBridges = bridgesMap.ibc.map((bridge) => bridge.bridgeAddresses).flat()
```

Comment:
1. Don't forget to filter out all disabled bridges (i.e. `!bridge.enabled`)
2. IBC chains do not have any bridge addresses. E.g.:
  ![Screenshot 2024-12-30 093439](https://github.com/user-attachments/assets/6dc087b2-9f06-4a62-b75c-d14d9814ed45)
  Maybe you can use the IBC chain id, i.e. `bridge.chain_id_name`